tests/test_detect_target.py

```diff
@@ -10,15 +11,58 @@
 
 from modules.detect_target import detect_target
 from modules import image_and_time
+from modules import detections_and_time
 
 
 DEVICE =                        0 if torch.cuda.is_available() else "cpu"
-MODEL_PATH =                    "tests/model_example/yolov8s_ultralytics_pretrained_default.pt"
-OVERRIDE_FULL =                 False  # Tests are able to handle both full and half precision.
-IMAGE_BUS_PATH =                "tests/model_example/bus.jpg"
-IMAGE_BUS_ANNOTATED_PATH =      "tests/model_example/bus_annotated.png"
-IMAGE_ZIDANE_PATH =             "tests/model_example/zidane.jpg"
-IMAGE_ZIDANE_ANNOTATED_PATH =   "tests/model_example/zidane_annotated.png"
+MODEL_PATH =                    pathlib.Path("tests", "model_example", "yolov8s_ultralytics_pretrained_default.pt")
+OVERRIDE_FULL =                 not torch.cuda.is_available()  # CPU does not support half precision
+IMAGE_BUS_PATH =                pathlib.Path("tests", "model_example", "bus.jpg")
+BOUNDING_BOX_BUS_PATH =         pathlib.Path("tests", "model_example", "bounding_box_bus.txt")
+IMAGE_ZIDANE_PATH =             pathlib.Path("tests", "model_example", "zidane.jpg")
+BOUNDING_BOX_ZIDANE_PATH =      pathlib.Path("tests", "model_example", "bounding_box_zidane.txt")
```

Comment:
Add `TEST_PATH` constant for tests/model_example/ directory, similar to `generate_expected.py` .

Also reduce the number of space characters between assignment operator and right side.